### PR TITLE
Add benchmarks for storage drivers

### PR DESCRIPTION
**Pull request by efectn**
closes https://github.com/gofiber/storage/issues/968

### CODE REVIEW

File: .github/workflows/benchmark.yml
Comment (gaby) [inline .github/workflows/benchmark.yml:132]: All the other DB's are not using TLS, we should remove it for the Benchmark of Redis and Ruedis.

TLS is known to cause performance issues for Redis in terms of Op/sec

**Review by gaby — CHANGES_REQUESTED** [review]

File: .github/workflows/benchmark.yml
Comment (gaby) [reply .github/workflows/benchmark.yml:132]: Seems the TestStore doesnt use TLS, so we dont need to do all this setup for the Benchmarks

**Review by gaby — APPROVED** [review]

**Comment by gaby** [comment]
May be good to wait until #993 is merged, but overall LGTM 💪

**Review by ReneWerner87 — APPROVED** [review]

**Pull request merged**
